qcfractal/cli/tests/test_cli.py

```diff
@@ -70,28 +71,79 @@ def test_manager_executor_manager_boot_from_file(active_server, tmp_path):
     assert testing.run_process(args, interupt_after=7, **_options)
 
 
+def cli_manager_runs(config_data, tmp_path):
+    temp_config = tmp_path / "temp_config.yaml"
+    temp_config.write_text(yaml.dump(config_data))
+    args = ["qcfractal-manager", f"--config-file={temp_config}", "--test"]
+    assert testing.run_process(args, **_options)
+
+
+def load_manager_config(adapter, scheduler):
+    config = read_config_file(os.path.join(_pwd, "manager_boot_template.yaml"))
+    config["common"]["adapter"] = adapter
+    config["cluster"]["scheduler"] = scheduler
+    return config
+
+
+@testing.mark_slow
+@testing.using_dask_jobqueue
+@testing.using_parsl
```

Comment:
One last thing here is that it requires both Parsl and Dask to be present. I think you can do:

```
testing.using_dask_jobqueue("dask", "slurm"),
```

If not can you remove the above decorators and within the test_cli_managers function use a 
```
if (adapter == "dask") and not testing.has_module("Dask"):
    pytest.mark.skip("reasons)
```